osu.Desktop/osu.Desktop.csproj

```diff
@@ -4,8 +4,8 @@
     <OutputType>WinExe</OutputType>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Description>A free-to-win rhythm game. Rhythm is just a *click* away!</Description>
-    <AssemblyName>osu!</AssemblyName>
-    <Title>osu!</Title>
+    <AssemblyName>osu!Tournament</AssemblyName>
+    <Title>osu!Tournament</Title>
```

Comment:
Seemingly we directly changed the name of osu!lazer executable.

Although not suggested in normal development, it would work as a temporary solution to the conflict with osu!stable.

Reply:
Just use this as a workaround, aim to verify if the gosu issue is caused by process name clash.